WalletWasabi/WabiSabi/Client/BobClient.cs

```diff
@@ -29,5 +29,28 @@ public async Task RegisterOutputAsync(Money amount, Script scriptPubKey, IEnumer
 				vsizeCredential,
 				cancellationToken).ConfigureAwait(false);
 		}
+
+		public async Task<(Credential[] RealAmountCredentials, Credential[] RealVsizeCredentials)> ReissueCredentialsAsync(
+			long amount1,
+			long amount2,
+			long vsize1,
+			long vsize2,
```

Comment:
i still don't think this is doing us any good over `IEnumerable`, but can't fix it without fixing `ArenaClient`

Reply:
ACK!